src/modules/about-us/OurValues/ourValues.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+import Image from "next/image";
+import { ourValuesList } from "./valuesHelper";
+
+type ourValuesProps = {
+  name: string;
+  description: string;
+  imageurl: string;
+};
```

Comment:
This (`ourValuesProps`) is not actually being used as component props here. Component props are a way to define the properties that the component receives like title, id etc. for the component (OurValues) here and should be named `OurValuesProps` with capital `O`. Either you can rename it (i.e. don't put `Props` at the end) if it's used for something else or the way I see it being used right now I don't think it's needed, you can probably remove it.